.circleci/config.yml

```diff
@@ -1,6 +1,7 @@
 version: 2.1
 orbs:
   node: circleci/node@3.0.0
+  ship: auth0/ship@0.3.0
 jobs:
   test:
     executor:
```

Comment:
this can make use of our `ship/node` executor, which by default uses image version `16.6.0`. Using those defaults you can remove the extra params:

```
executor: ship/node
```

If you want to change that to use specifically 15.x, you'd only need to change the param `name` to `ship/node` and keep the `tag` parameter the same. I just verified and that version has the minimum required **npm** version installed, as per this library's notice on top of the readme.

![image](https://user-images.githubusercontent.com/3900123/132694840-38fae7af-aad2-48ac-ad85-5b19f058ee48.png)